test/functional/p2p_disconnect_ban.py

```
@@ -74,24 +77,33 @@
         self.nodes[1].setban("2001:4d48:ac57:400:cacf:e9ff:fe1d:9c63/19", "add", 1000)  # ban for 1000 seconds
         listBeforeShutdown = self.nodes[1].listbanned()
         assert_equal("192.168.0.1/32", listBeforeShutdown[2]['address'])
-        self.bump_mocktime(2)
-        self.wait_until(lambda: len(self.nodes[1].listbanned()) == 3, timeout=10)
+
+        self.log.info("setban: test banning with absolute timestamp")
+        self.nodes[1].setban("192.168.0.2", "add", self.mocktime + 120, True)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)